test/e2e/specs/wp-inline-help-support-search-spec.js

```diff
@@ -28,49 +29,56 @@ before( async function () {
 	driver = await driverManager.startBrowser();
 } );
 
-describe( `[${ host }] Inline Help support search: (${ screenSize }) @parallel`, async function () {
+describe( `[${ host }] Inline Help: (${ screenSize }) @parallel`, async function () {
 	this.timeout( mochaTimeOut );
 
 	step( 'Login and select a non My Home page', async function () {
 		const loginFlow = new LoginFlow( driver );
 
-		// The "inline help" FAB should not appear on the My Home
-		// because there is already a support search "Card" on that
-		// page. Therefore we select the "Themes" page for our tests.
-		await loginFlow.loginAndSelectThemes();
+		await loginFlow.loginAndSelectMySite();
 
 		// Initialize the helper component
-
 		inlineHelpPopoverComponent = await InlineHelpPopoverComponent.Expect( driver );
 	} );
 
-	describe( 'Popover UI visibility and interactions', function () {
-		step( 'Check help toggle is visible', async function () {
-			await inlineHelpPopoverComponent.isToggleVisible();
+	describe( 'Popover UI visibility', function () {
+		step( 'Check help toggle is not visible on My Home page', async function () {
+			const sidebarComponent = await SidebarComponent.Expect( driver );
+			await sidebarComponent.selectMyHome();
+
+			// The toggle only hides once the "Get help" card is rendered so
+			// we need to give it a little time to be removed.
+			await inlineHelpPopoverComponent.waitForToggleNotToBePresent();
 		} );
 
-		step( 'Open Inline Help popover', async function () {
-			await inlineHelpPopoverComponent.toggleOpen();
+		step( 'Check help toggle is visible on Theme page', async function () {
+			const sidebarComponent = await SidebarComponent.Expect( driver );
 
-			const isPopoverVisible = await inlineHelpPopoverComponent.isPopoverVisible();
-			assert.equal( isPopoverVisible, true, 'Popover was not opened correctly.' );
-		} );
+			// The "inline help" FAB should not appear on the My Home
+			// because there is already a support search "Card" on that
+			// page. Therefore we select the "Themes" page for our tests
+			// and then we will switch away to the Homepage to verify
+			// that the Inline Help is not shown
+			await sidebarComponent.selectThemes();
 
-		step( 'Close Inline Help popover', async function () {
-			await inlineHelpPopoverComponent.toggleClosed();
-			const isPopoverVisible = await inlineHelpPopoverComponent.isPopoverVisible();
-			assert.equal( isPopoverVisible, false, 'Popover was not closed correctly.' );
+			// Once removed we can assert is is invisible.
+			const isToggleVisible = await inlineHelpPopoverComponent.isToggleVisible();
+			assert.equal(
+				isToggleVisible,
+				true,
+				'Inline Help support search was not shown on Theme page.'
+			);
 		} );
 	} );
 
 	describe( 'Performing searches', function () {
```

Comment:
@getdave I feel like almost everything in this section could get removed since it's tested more thoroughly in `test/e2e/specs/wp-my-home-support-search-spec.js`. Maybe leave a basic "Can perform a search" and "Can remove the search" to smoke test that it's still working correctly within the popover?